### PR TITLE
[1.11] Bump Go version to 1.5.4 (security fix)

### PR DESCRIPTION
**Pull request by tiborvass**
Security fix for Go:

https://groups.google.com/forum/#!msg/golang-announce/9eqIHqaWvck/kXsfO0ogLAAJ

Related to #21978

### CODE REVIEW

**Comment by icecrime** [comment]
Triggered ARM: LGTM if green :+1:

File: Dockerfile.armhf [hunk 1]
```diff
@@ -65,7 +65,7 @@ RUN cd /usr/local/lvm2 \
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
 # Install Go
-ENV GO_VERSION 1.5.3
+ENV GO_VERSION 1.5.4
```
Comment (tonistiigi) [inline Dockerfile.armhf]: arm sources probably don't update that quickly.

**Comment by tonistiigi** [comment]
LGTM

**Comment by thaJeztah** [comment]
Looks like you missed some; https://github.com/docker/docker/search?p=2&q=GO_VERSION&type=Code&utf8=✓

**Comment by thaJeztah** [comment]
Created a PR for the bump branch to fix the missing ones; https://github.com/docker/docker/pull/21987

**Pull request merged**